README.md

```diff
@@ -41,6 +42,11 @@ https://github.com/games647/LambdaAttack/releases
 
 ![Gui](https://i.imgur.com/6U00ZwA.png)
 
+## Command Line Usage
+LambdaAttack can be used from the command line without a GUI.  
+To get a list of all available options, run
+`java -jar lambdaattack.jar --help`.
+
```

Comment:
Maybe it's useful to document the parameters. Then people know how far tool is configurable without testing it.

Reply:
Yeah, that crossed my mind as well, and I agree. I was simply too lazy to list them all, but I'll add that in a sec